archive/rest-catalog/brands.v2.yaml

```diff
@@ -0,0 +1,515 @@
+
+  termsOfService: 'https://www.bigcommerce.com/terms'
+  contact:
+    name: BigCommerce
+    url: 'https://www.bigcommerce.com'
+    email: support@bigcommerce.com
+  version: '1.0.1'
+servers:
+  - url: 'https://api.bigcommerce.com/stores/{store_hash}/v2'
+    variables:
+      store_url:
+        default: store_url
+        description: Permanent url of the BigCommerce store.
+    description: BigCommerce API Gateway
+security:
+  - basicAuth: []
+tags:
+  - name: Brands V2
+paths:
+  '/brands':
+    get:
+      tags:
+        - Brands V2
+      summary: Gets list of Brands.
+      description: Gets the collection of Brands. (Default sorting is by brand id, from lowest to highest.)
+      operationId: getBrands
+      parameters:
+        - $ref: '#/components/parameters/Name'
+        - $ref: '#/components/parameters/IdMin'
+        - $ref: '#/components/parameters/IdMax'
+        - $ref: '#/components/parameters/Page'
+        - $ref: '#/components/parameters/Limit'
+      responses:
+        '200':
+          $ref: '#/components/responses/brands'
+        '400':
+          $ref: '#/components/responses/errors'
+        '401':
+          $ref: '#/components/responses/errorsWithDetails'
+        '403':
+          $ref: '#/components/responses/errorsWithDetails'
+    post:
+      tags:
+        - Brands V2
+      summary: Create a Brand.
+      description: |-
+        Creates a *Brand*.
+
+        **Required Fields**
+        
+        The following properties of the brand are required. The request won’t be fulfilled unless these properties are valid.
+        - name
+        
+        **Read-only Fields**
+        
+        The following properties of the brand are read-only. If one or more of these properties are included in the request, it will be rejected.
+        - id
+        
+        **Limits**
+        - 30 000 brands per store limit
+        To maximize system performance, BigCommerce caps the number of brands that can be added to a store at 30,000. If your POST causes the store to exceed the maximum of 30,000 brands, BigCommerce will return a 403 error.
+      operationId: createBrand
+      requestBody:
+        content:
+          'application/json':
+            schema:
+              $ref: '#/components/schemas/BrandPost'
+          'application/xml':
+            schema:
+              $ref: '#/components/schemas/BrandPost'
+        required: true
+      responses:
+        '201':
+          $ref: '#/components/responses/brand'
+        '400':
+          $ref: '#/components/responses/errors'
+        '401':
+          $ref: '#/components/responses/errorsWithDetails'
+        '403':
+          $ref: '#/components/responses/errorsWithDetails'
+        '409':
+          $ref: '#/components/responses/errorsWithDetails'
+    delete:
+      tags:
+        - Brands V2
+      summary: Deletes all existing on store brands.
```

Comment:
```suggestion
      summary: Deletes all existing on store brands. (Deprecated)
```

Reply:
Changed to "summary: Deletes all existing on store brands. (Deprecated)"